CMakeLists.txt

```diff
@@ -150,7 +150,7 @@ if (WITH_TESTS)
     enable_testing()
 endif()
 
-add_subdirectory(plugins)
+# add_subdirectory(plugins)
```

Comment:
Cannot find arbiter, it is because i include the `Connector.hpp` to the reader class. The plugins do not seem to have access to `#include <nlohmann/json.hpp>` and `#include <arbiter/arbiter.hpp>`